screencap.py

```diff
@@ -119,46 +134,40 @@ def retranslateUi(self, mainWindow):
         self.actionmvk.setText(_translate("mainWindow", "mvk"))
 
     def clicked_format(self, extension):
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        f.close()
-        f = open('memory_screenshot.txt','w')
-        lst[1] = extension
-        f.write('{0},{1},{2}'.format(lst[0],lst[1],lst[2]))
-        f.close()
+        if config['image']['extension'] != extension:
+            config['image']['extension'] = extension
+            self.save_config()
 
     def clicked_folder(self):
-        self.screenshot_path = QtWidgets.QFileDialog.getExistingDirectory(None, 'Select Folder','C:/Users')
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        f.close()
-        f = open('memory_screenshot.txt','w')
-        lst[2] = self.screenshot_path
-        f.write('{0},{1},{2}'.format(lst[0],lst[1],lst[2]))
-        f.close()
-
+        screenshot_path = QtWidgets.QFileDialog.getExistingDirectory(None, 'Select Folder')
+        if config['image']['path'] != screenshot_path:
+            config['image']['path'] = screenshot_path
+            self.save_config()
+            
     def clicked_sceenshot(self):
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        i = int(lst[0])+1
-        i = str(i)
-        path  = lst[2] + '/screenshot' + i + '.' + lst[1]
-        f.close()
+        if (config['image']['path']) == " ":
```

Comment:
If the value will be None, the condition can be simplified because `bool(None) == False`
```suggestion
        if (config['image']['path']):
```

Reply:
if the path is None, I need a condition to be true for the user to select a folder 
```suggestion
        if (config['image']['path']) == None
```